hbase-migration-tools/bigtable-hbase-replication/README.md

```diff
@@ -111,6 +111,10 @@ classpath.
 </property>
 ```
 
+We recommend specifying an application profile by setting config key 
+`google.bigtable.app_profile.id`. A single-cluster routing application profile
+preserves order of mutations between HBase and Cloud Bigtable. 
```

Comment:
Can you link to https://cloud.google.com/bigtable/docs/configuring-app-profiles?